addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -66,10 +66,23 @@ def _l10n_it_edi_check_invoice_configuration(self, invoice):
 
         return errors
 
+    def _l10n_it_edi_is_reverse_charge(self, invoice):
+        company_country = invoice.company_id.country_id
+        partner_country = invoice.commercial_partner_id.country_id
+        return (
+            invoice.is_outbound()
+            and invoice.company_id == self.env.company
```

Comment:
does not make any sense

Reply:
Oh, ok, think I can remove this.
Even though I think this doesn't change much.
I'll already probably be exporting invoices for my company, right?

Reply:
If you have multiple companies active, you could export for another company then self.env.company.